### PR TITLE
Proper formatting support

### PR DESCRIPTION
**Pull request by yukimono**
Through [Maruku-dirty](https://github.com/skiprope/maruku-dirty).

The handler is untested though.

### CODE REVIEW

File: config/initializers/maruku.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+require 'maruku-dirty' # ?????? ??
```
Comment (nilsding) [inline config/initializers/maruku.rb]: maruku-dirty is automatically `require`d if it's listed in the Gemfile

**Comment by yukimono** [comment]
I was unaware of that lmao

**Comment by nilsding** [comment]
I'll take a look at Maruko and will then decide if I'll merge it or not.

**Comment by TheHowl** [comment]
Why was this closed?

**Comment by yukimono** [comment]
because i'm rewriting it, see Retrospring/bugs#72

**Pull request closed without merging**